src/sentry/integrations/base.py

```diff
@@ -125,6 +125,53 @@ class IntegrationFeatures(Enum):
     DEPLOYMENT = "deployment"
 
 
+# Integration Types
```

Comment:
hmm i would love to do something so that when you register the integrations here https://github.com/getsentry/sentry/blob/c92f4b80d9cfa5c8762751fdb7923d63e61565d1/src/sentry/runner/initializer.py#L59-L76

you also register the integration type... cc @RyanSkonnord if you have thoughts